WebDriverAgentLib/Utilities/FBBaseActionsSynthesizer.h

```diff
@@ -15,6 +15,15 @@
 NS_ASSUME_NONNULL_BEGIN
 
 #if !TARGET_OS_TV
+
+/**
+ Xcode SDKs 11 to 13.1 (including) have known bug that makes xctest to not properly handle event coordinates
+ for non-portrait orientations.
+ 
+ @returns YES if the current platform has the above issue.
+ */
+BOOL FBShouldWorkaroundCoordinatesTranslationBug(void);
```

Comment:
Let me check on a real device as well.
So far, I've confirmed weird behaviors only on simulators.

Reply:
It seems real devices still need this.
With this change, clicking an element via W3C action (`@driver.action.click(el).perform`) after changing the orientation scenario did not work (the "click" did not click the expected element coordinate) while the current master can work.

Reply:
was xcode version the same in both cases?

Reply:
in theory we could make that condition even more complicated to only return false for simulators

Reply:
Yes, both were Xcode beta with this repository's integration test app.
I haven't checked them on simulator yet.

Reply:
mm, a simulator (iPad Pro 12.9 inch 5 gen x iOS 15.2, iPhone 13 x iOS 15.2) also had the same issue with this branch.
landscape case was not able to click an expected element with this branch. Xcode was beta as same as the above.

(https://github.com/appium/WebDriverAgent/pull/547 is with Xcode 13 and this branch. it got some gesture error in iPad. So, so for now, current change fixes probably `testPress` on iOS 15.2, but it causes landscape case's action errors)